dataclients/kubernetes/kube.go

```diff
@@ -898,6 +905,16 @@ func (c *Client) ingressToRoutes(state *clusterState) ([]*eskip.Route, error) {
 						}
 					}
 
+					// add pre-configured default filters
+					if defFilter, ok := defaultFiltersOf(prule.Backend.ServiceName, i.Metadata.Namespace, defaultFilters); ok {
+						defaultFilters, err := eskip.ParseFilters(defFilter)
+						if err != nil {
+							logger.Errorf("Can not parse default filters: %v", err)
```

Comment:
nitpick: "Can not" -> "Failed to"

Reply:
It's consistent to already implemented `Can not parse annotation filters` error (line 903).

Reply:
Change both, because else it would be `cannot` :smile_cat: